i18n/locales/ja/ch-applications/qaoa.ipynb

```diff
@@ -7090,7 +7090,7 @@
     " - 枝 $(k,l) \\in E$ に 角度 $\\gamma$のイジング・タイプ・ゲート $U_{k,l}(\\gamma)$ を6つ作用させます。Qiskitに備わっているゲートを用いて以下のように表現することができます。\n",
     "\n",
     "\n",
-    "$$ U_{k,l}(\\gamma) = CU1(-2\\gamma)_{k,l} U1(\\gamma)_k U1(\\gamma)_l$$\n",
+    "$$ U_{k,l}(\\gamma) = CU1(2\\gamma)_{k,l} U1(-\\gamma)_k U1(-\\gamma)_l$$\n",
```

Comment:
Following the notation indicated just above, the unitary operator should be:
U<sub>k,l</sub>(&gamma;) =e<sup> -i&gamma; &Sigma;(1 - Z<sub>k</sub>Z<sub>m</sub>)/2</sup>= CU1(2&gamma;)<sub>k,l</sub> U1(-&gamma;)<sub>k</sub> U1(-&gamma;)<sub>l</sub>